src/components/product/index.js

```diff
@@ -100,33 +100,43 @@ const Product = (props) => {
 
   const addToCart = async (item) => {
     try {
-      const user = localStorage.getItem("uid");
-      const cartRef = doc(db, "carts", user);
-      const productRef = doc(cartRef, "products", `${item.id}`);
+      const user = localStorage.getItem('uid');
+      const cartRef = doc(db, 'carts', user);
+      const productRef = doc(cartRef, 'products', `${item.id}`);
       await setDoc(productRef, { ...item, quantity: 1 });
       setIsadded(true);
       context.fetchCartProducts();
     } catch (error) {
-      console.error("Error adding item to cart:", error);
+      console.error('Error adding item to cart:', error);
     }
   };
 
   const addToWishlist = async (item) => {
-    console.log("addToWishlist");
+    console.log('addToWishlist');
```

Comment:
[As it was already there earlier as well]
could you remove this console.log() ? 
Seems unnecessary as the application do not need to see a message on console to see if the function is called or not.

Reply:
Done.